marklogic-data-hub-central/ui/e2e/cypress.json

```diff
@@ -3,12 +3,17 @@
     "video": true,
     "viewportWidth": 1680,
     "viewportHeight": 1050,
+    "numTestsKeptInMemory": 1,
     "defaultCommandTimeout": 10000,
     "reporter": "junit",
     "reporterOptions": {
       "mochaFile": "results/cypress-report[hash].xml",
       "toConsole": true
     },
+    "retries": {
+      "runMode": 2,
```

Comment:
I don't think we need this. The tests shouldn't fail if everything is reliable.

Reply:
I'll back out the retries configuration changes. 

Any strong feelings on the memory settings? 50 seems like a high default number and with the server running on the localhost if we add enough tests we could start running into resource contention issues.

Reply:
If it's individual e2e tests, we have already exceeded 50 and I don't think adjusting the number will make a difference.